source/http/source.go

```diff
@@ -92,9 +94,19 @@ func (hs *httpSource) Identifier(scheme, ref string, attrs map[string]string, pl
 				return nil, err
 			}
 			id.GID = int(i)
+		default:
+			if name, found := strings.CutPrefix(k, pb.AttrHTTPHeader); found {
+				id.Header = append(id.Header, HeaderField{Name: name, Value: v})
```

Comment:
`http. CanonicalHeaderKey(name)`

Reply:
This should be covered by the use of `req.Header.Set` later on but I can add it here too for posterity.

Reply:
Ah ok, not needed.